modules/tests/test_opendaq_device_modules/test_native_streaming_modules.cpp

```diff
@@ -142,3 +138,99 @@ TEST_F(NativeStreamingModulesTest, DISABLED_RenderSignal)
 
     std::this_thread::sleep_for(std::chrono::milliseconds(5000));
 }
+
+TEST_F(NativeStreamingModulesTest, Reconnect)
```

Comment:
Is there a possibility of this being split into smaller parts?

Reply:
Divided into two separate tests: one for validating data reading recovery and another for checking the equality of client signals before and after reconnection.